frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -820,11 +822,14 @@ const TestResultsList = () => {
         : filterFacilityId || activeFacilityId,
     ...queryParams,
   };
-
-  const count = useQuery(resultsCountQuery, {
+  const count = useGetResultsCountByFacilityQuery({
     fetchPolicy: "no-cache",
     variables: countQueryVariables,
   });
+  /*const count = useQuery(RESULTS_COUNT_QUERY, {
```

Comment:
This comment can probably be removed

Reply:
Thanks for catching this 😮 !!! I removed it